docs/mining/overview.md

```diff
@@ -26,74 +26,141 @@ transactions waiting to be processed in the mempool.
 Decred uses the [BLAKE-256](../research/blake-256-hash-function.md) hashing function. PoW mining can be done using GPUs. However, ASICs that mine DCR are now available, and since their introduction the PoW difficulty has become high enough to make GPU
 mining unlikely to be profitable.
 
----
+## Purchasing specialized hardware (ASIC)
```

Comment:
Capitalize the header and include a horizontal rule before it please.

```
---

## Purchasing Specialized Hardware (ASIC)
```

Reply:
Fixed